Apps/W1/HybridGP/app/src/Migration/GPTables/GPGL00105.Table.al

```diff
@@ -0,0 +1,46 @@
+table 41000 "GP GL00105"
+{
+    DataClassification = CustomerContent;
+    fields
+    {
+        field(1; ACTINDX; Integer)
+        {
+            DataClassification = CustomerContent;
+        }
+        field(2; ACTNUMBR_1; Text[9])
+        {
+            DataClassification = CustomerContent;
+        }
+        field(3; ACTNUMBR_2; Text[9])
+        {
+            DataClassification = CustomerContent;
+        }
+        field(4; ACTNUMBR_3; Text[9])
+        {
+            DataClassification = CustomerContent;
+        }
+        field(5; ACTNUMBR_4; Text[9])
+        {
+            DataClassification = CustomerContent;
+        }
+        field(6; ACTNUMBR_5; Text[9])
```

Comment:
Be sure to understand that this table (GP00105) can have a different table structure depending upon how the customer setup GP initially. The customer's table may have more or less than 5 segments (ACTNUMBR_6, ACTNUMBR_7, ACTNUMBR_8). I thought we coded prior such tables with 8 segments to account for most GP customers (I don't recall the reason behind the 8-segment number). I would need to revisit those migration tables to be sure though. Nikola had one or two tables like this when he removed tables from the extension and the replication service. Just not sure how you arrived at 5 segments.

Reply:
I used this. https://github.com/microsoft/BCTech/blob/master/samples/CloudMigration/GenerateALTablesFromSQLSchema/GPTablesPTE/GPTables/MSFTGL00105.Table.al

Is there a max number of segments? It wouldn't hurt to add the max here.

Reply:
I was looking at tables (GPGL00100, GPGL10110, etc.) in the GP migration extension which use an ACTNUMBR_x field. they all appear to be using 8 segments instead of 5. The maximum allowed by GP is not being used. I don't remember why. I would suggest using 8.